### DISCUSSION TITLE
Assets optimization and Content-Encoding

### DISCUSSION THREAD

**Discussion by Harkonnen13** [nuxt/nuxt#17398] [Questions]
Hello everyone.
My project is pretty big, and i am thinking about optimizations right now.
I use vite compression plugin, and generate gzip/brotli files for my assets. But i can't find info to setup routes to use that.
Is there any way to make it work?

**Comment by qnp** [comment]
Hello,
on this topic:
- Original related discussion => https://github.com/nuxt/framework/discussions/3472
- Issue open on nitro => https://github.com/unjs/nitro/issues/69
